internal/configs/sharding/localnet.go

```diff
@@ -11,6 +11,20 @@ import (
 	"github.com/harmony-one/harmony/internal/genesis"
 )
 
+var (
+	localnetReshardingEpoch = []*big.Int{
+		big.NewInt(0), big.NewInt(localnetV1Epoch), params.LocalnetChainConfig.StakingEpoch, params.LocalnetChainConfig.TwoSecondsEpoch,
+	}
+	// Number of shards, how many slots on each , how many slots owned by Harmony
+	localnetV0   Instance
+	localnetV1   Instance
+	localnetV2   Instance
+	localnetV3   Instance
+	localnetV3_1 Instance
+	localnetV3_2 Instance
+	localnetV4   Instance
```

Comment:
what happened and what required changes will we have to do if we decide to have a completely different name and network config here ?

Reply:
These are the original names and wasn't touched by this PR.